src/components/Garden/Feed/ProposalSupport.tsx

```diff
@@ -112,6 +139,7 @@ function DecisionSummaryBar({ proposal }) {
             css={`
               margin-left: ${1 * GU}px;
             `}
+            token={null}
```

Comment:
Its necessary? Looking on source code looks its okay be removed. WDYT?

Reply:
<img width="854" alt="Captura de ecrã 2022-01-22, às 18 45 02" src="https://user-images.githubusercontent.com/13184907/150651497-b3465353-8d58-49c9-827d-b9f0b87bd539.png">

Reply:
And about 
![image](https://user-images.githubusercontent.com/691510/150651801-2bc341ed-9f09-4419-854a-97df2b0f0c76.png)

It should solve and make more sense with the inner code of SummaryRow

Reply:
Yes it should. But I didn't want to change any more files. Thats why the `token={null}` seems fine to me.